src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/TelemetryReporter.ts

```diff
@@ -10,6 +10,7 @@ export class TelemetryReporter {
     private readonly razorExtensionActivated = new TelemetryEvent('VSCode.Razor.RazorExtensionActivated');
     private readonly debugLanguageServerEvent = new TelemetryEvent('VSCode.Razor.DebugLanguageServer');
     private readonly workspaceContainsRazorEvent = new TelemetryEvent('VSCode.Razor.WorkspaceContainsRazor');
+    private readonly blazorWasmDebugStartedEvent = new TelemetryEvent('VSCode.Razor.BlazorWASMDebugger.Started');
```

Comment:
You'll need to make sure to classify this telemetry properly in GDPR: https://microsoft.sharepoint.com/teams/DD_VSPlat/Diagnostics/_layouts/15/Doc.aspx?sourcedoc={57593dcb-cb9c-4761-9981-a048d9896671}&action=edit&wd=target%282.%20KB%2FProcess%20Reference.one%7C7caa1968-9bcb-468b-9f8b-530a6ab088ce%2FGDPR%20telemetry%20property%20registration%7C9bfb358e-087d-4bc9-b852-f1464ef93b59%2F%29